sql/mob_skill_lists.sql

```diff
@@ -3849,7 +3849,9 @@ INSERT INTO `mob_skill_lists` VALUES ('Zmey_Gorynych',2001,380); -- scythe_tail
 
 INSERT INTO `mob_skill_lists` VALUES ('Copper_Borer',2002,1817); -- thermal_pulse
 
--- Next available ID: 2003
+INSERT INTO `mob_skill_lists` VALUES ('Gharial',2003,2101); -- demoralizing_roar
```

Comment:
2003 got used by a pr merged earlier today I think.

Reply:
correction: I'd forgotten to hit the button https://github.com/LandSandBoat/server/pull/5146/files#diff-28d0cbb5981fc0487c120848c5354b354cf55784208e97a2483deb3acbc64c79R3852